kyuubi-hive-jdbc/src/main/java/org/apache/kyuubi/jdbc/hive/KyuubiConnection.java

```diff
@@ -937,6 +943,17 @@ public void close() throws SQLException {
     }
   }
 
+  public void closeOnLaunchEngineFailure(Thread logThread) throws SQLException {
+    if (logThread != null && logThread.isAlive()) {
+      logThread.interrupt();
+      try {
+        logThread.join(DEFAULT_ENGINE_LOG_THREAD_TIMEOUT);
+      } catch (Exception e) {
+      }
```

Comment:
Does the `logThread` need to set to null?

Reply:
addressed, thanks